{{cookiecutter.project_slug}}/setup.py

```diff
@@ -68,5 +69,6 @@
     ],
     extras_require={
         'dev':  ['prospector[with_pyroma]', 'yapf', 'isort'],
-    }
+    },
+    data_files=[('citation/{{ cookiecutter.project_slug.lower().replace(' ', '_').replace('-', '_')}}', ['CITATION.cff'])]
```

Comment:
Maybe it would be safer to use MANIFEST.in to include the file?

Reply:
Do files in `MANIFEST.in` end up in a standard directory?

Reply:
I am not sure the CITATION.cff file included in the MANIFEST.in file is available after installation.

I've pushed two branches using the data-file and the manifest file options to test. So I can create a python module (module1) like this:

```
cookiecutter https://github.com/NLeSC/python-template.git --checkout data-file-citation
pip install module1/
```

Then in python:
```python
import module1
module1.__version__
```

And that works, but if I do this one:
```
cookiecutter https://github.com/NLeSC/python-template.git --checkout with-manifest
pip install module2/
```
Then in python:
```python
import module2
module2.__version__
```
It breaks. Any ideas how to fix this? What am I doing wrong?

Reply:
It looks like you're encountering this issue https://github.com/pypa/setuptools/issues/1691

Reply:
Yes, looks like a similar issue. Shall we go with the solution in `data-file-citation` branch, since it seems to be working? It is also the same which we are doing in [xenon](https://github.com/xenon-middleware/pyxenon/blob/master/setup.py#L52-L55).

Reply:
Not sure if this is really an issue, but does that work with multiple versions of python installed? 

Because that installs the python code in `lib/python3.8/site-packages/my_python_project/`, while the citation file is installed in `citation/my_python_project/` (without the python3.8 directory).

Reply:
Yes, good point. I guess we run the risk of overwriting our own citation files for different versions of python (if you have different versions of the package for different versions of python). But I guess it is a fairly small chance of happening?

Reply:
It's up to you what you want to decide on this Carlos. I'm not sure I would recommend doing it like this, but I also don't know any better way to do it.

Reply:
@fdiblen -- when you have time, could you have a look. Just want to see if you have an alternative and if you think it has a high risk of breaking.

Reply:
Maybe using [versioneer](https://github.com/warner/python-versioneer) could be an alternative to installing the citation file?

Reply:
I just came across another suggestion to solving the problem that it's not possible to include a file from the top level directory via MANIFEST.in: create a symbolic link to the file inside the package directory. That way it should get copied: https://stackoverflow.com/questions/8556996/setuptools-troubles-excluding-packages-including-data-files/11669299#11669299